wallet/reference/non-evm-apis/starknet-snap-api.md

```diff
@@ -0,0 +1,867 @@
+  `REJECTED` state.
+  The default is `false`.
+- `chainId`: `string` - (Optional) ID of the target Starknet network.
+  The default is the Starknet Sepolia testnet.
+
+#### Returns
+
+The list of the transaction records.
+
+#### Example
+
+<Tabs>
+<TabItem value="Request">
+
+```js
+await getEip6963Provider.request({
+  method: "wallet_invokeSnap",
+  params: {
+    snapId: "npm:@consensys/starknet-snap",
+    request: {
+      method: "starkNet_getTransactions",
+      params: {
+        senderAddress: "0xb60e8dd61c5d32be8058bb8eb970870f07233155",
+        contractAddress: "0x5B38Da6a701c568545dCfcB03FcB875f56beddC4",
+        pageSize: 25,
+        txnsInLastNumOfDays: 7,
+        withDeployTxn: true,
+        onlyFromState: false,
+        "chainId": "0x534e5f5345504f4c4941"
+      },
+    },
+  },
+})
+```
+
+</TabItem>
+<TabItem value="Result">
+
+```json
+[
+  {
+    "txnHash": "0x05a56e2d52c817161883f50c441c3228cfe54d9f84b5b5b8b1c8b8e0e6f7e6d8",
+    "txnType": "invoke",
+    "senderAddress": "0xb60e8dd61c5d32be8058bb8eb970870f07233155",
+    "contractAddress": "0x5B38Da6a701c568545dCfcB03FcB875f56beddC4",
+    "contractFuncName": "transfer",
+    "contractCallData": ["0x789...", "100"],
+    "status": "ACCEPTED_ON_L2",
+    "failureReason": null,
+    "eventIds": ["0xdef..."],
+    "timestamp": 1234567890,
+    "chainId": "0x534e5f5345504f4c4941"
+  },
+  // ... more transactions
+]
+```
+
+</TabItem>
+</Tabs>
+
+### `starkNet_getTransactionStatus`
+
+Gets the status of a transaction.
+
+#### Parameters
+
+- `transactionHash`: `string` - Hash of the target transaction.
+- `chainId`: `string` - (Optional) ID of the target Starknet network.
+  The default is the Starknet Sepolia testnet.
+
+#### Returns
+
+The [status](https://docs.starknet.io/architecture-and-concepts/network-architecture/transaction-life-cycle/)
+of the transaction.
+
+#### Example
+
+<Tabs>
+<TabItem value="Request">
+
+```js
+await getEip6963Provider.request({
+  method: "wallet_invokeSnap",
+  params: {
+    snapId: "npm:@consensys/starknet-snap",
+    request: {
+      method: "starkNet_getTransactionStatus",
+      params: {
+        transactionHash: "0x05a56e2d52c817161883f50c441c3228cfe54d9f84b5b5b8b1c8b8e0e6f7e6d8",
+        "chainId": "0x534e5f5345504f4c4941"
+      },
+    },
+  },
+})
+```
+
+</TabItem>
+<TabItem value="Result">
+
+```json
+"ACCEPTED_ON_L2"
+```
+
+</TabItem>
+</Tabs>
+
+### `starkNet_getValue`
+
+Calls a `VIEW` method on any contract.
+
+#### Parameters
+
+- `contractAddress`: `string` - Address of the target contract.
+- `contractFuncName`: `string` - Target function name of the contract.
+- `contractCallData`: `string` - (Optional) Call data for the target function with `,` as a separator.
+- `chainId`: `string` - (Optional) ID of the target Starknet network.
+  The default is the Starknet Sepolia testnet.
+
+#### Returns
+
+The response from Starknet's `feeder_gateway/call_contract` API endpoint.
```

Comment:
Since the `feeder_gateway`/`gateway` has been [deprecated](https://community.starknet.io/t/feeder-gateway-deprecation/100233), should this be updated?